Automattic/sensei
# Add lesson actions preview

Fixes #3821

### Changes proposed in this Pull Request

* It adds the lesson actions preview states.

### Testing instructions

* Go to the lesson editor.
* Add the _Lesson actions_ block.
* Change the preview state in the toolbar, and make sure the buttons receive a new opacity based on the selected state.
* Click on the buttons, and make sure the state is updated automatically to the selected button respective preview state.


### Screenshot / Video

https://user-images.githubusercontent.com/876340/105210912-6e2b0800-5b2a-11eb-8d80-965a6c5bce8d.mov
